portfolio/src/main/webapp/script.js

```diff
@@ -70,3 +70,15 @@ function contact() {
 		</ul>`; 
 	console.log('Contact!'); 
 }
+
+function getComments() {
+    console.log('Hello!');
```

Comment:
nit - match the indentation here

Reply:
hm I don't see any odd indentations on my side...I will push and then look at it in the github UI!